### ISSUE TITLE
Why not using welford algorithm in batchnorm (cpu)?

### ISSUE THREAD

**Issue by FateScript** [pytorch/pytorch#69525]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

Hello, Pytorch team,

I am reading your code of batchnorm (cpu version) shown below,
https://github.com/pytorch/pytorch/blob/bceb1db885cafa87fe8d037d8f22ae9649a1bba0/aten/src/ATen/native/cpu/batch_norm_kernel.cpp#L189-L227

I notice that your implementation access memory twice, and I'm wondering is there any consideration for not using welford algorithm?

Thanks.

cc @VitalyFedyunin @ngimel

**Comment by mrshenli** [comment]
cc @albanD @gchanan in case you know the history :)

**Comment by albanD** [comment]
I think the main reason is that this is the "slow" generic cpu implementation. So there was never great efforts to speed this up as most users never en up executing this code but use the cudnn/mkldnn version of this op.

**Comment by FateScript** [comment]
Ok, thanks @albanD  @mrshenli . Is there any contribution needed or just close this issue ?

**Comment by albanD** [comment]
I think we would accept a PR speeding this up, cc @ngimel  to confirm this?
